SQLHomework1.sql

```diff
@@ -0,0 +1,193 @@
+
+SELECT *
+FROM Customers;
+
+INSERT INTO Orders
+VALUES ('Order july', 4);
+INSERT INTO Orders
+VALUES ('Order september', 1);
+INSERT INTO Orders
+VALUES ('Order June', 4);
+INSERT INTO Orders
+VALUES ('order may', 2);
+INSERT INTO Orders
+VALUES ('order may', 3);
+INSERT INTO Orders
+VALUES ('order december', 1);
+INSERT INTO Orders
+VALUES ('order may', 5);
+INSERT INTO Orders
+VALUES ('order march', 6);
+
+
+SELECT *
+FROM Orders;
+
+INSERT INTO Products
+VALUES ('PEACH');
+INSERT INTO Products
+VALUES ('APRICOTS');
+INSERT INTO Products
+VALUES ('PLUMS');
+INSERT INTO Products
+VALUES ('CHERRIES');
+INSERT INTO Products
+VALUES ('BLUEBERRIES');
+INSERT INTO Products
+VALUES ('BLACKBERRIES');
+
+SELECT * 
+FROM Products;
+
+INSERT INTO Orders_Products
+VALUES (1, 6);
+INSERT INTO Orders_Products
+VALUES (1, 2);
+INSERT INTO Orders_Products
+VALUES (2, 4);
+INSERT INTO Orders_Products
+VALUES (3, 1);
+INSERT INTO Orders_Products
+VALUES (6, 1);
+INSERT INTO Orders_Products
+VALUES (5, 2);
+INSERT INTO Orders_Products
+VALUES (3, 4);
+INSERT INTO Orders_Products
+VALUES (3, 1);
+
+SELECT * 
+FROM Orders_Products;
+
+--Exercise 2. Display all the Customers that are older than 15.
+SELECT * 
+FROM Customers
+WHERE Age > 15;
+
+--Exercise 3. Display the total number of Addresses.
+SELECT COUNT(Id) As NumberOfAddresses
+FROM Addresses;
+
+--Exercise 4. Update the name of each Product to some other value.
+UPDATE Products
+SET Name = 'Beans'
+WHERE Id = 1;
+UPDATE Products
+SET Name = 'Eggplant'
+WHERE Id = 2;
+UPDATE Products
+SET Name = 'Mushrooms'
+WHERE Id = 3;
+UPDATE Products
+SET Name = 'Spinach'
+WHERE Id = 4;
+UPDATE Products
+SET Name = 'Potato'
+WHERE Id = 5;
+UPDATE Products
+SET Name = 'Lentils'
+WHERE Id = 6;
+
+--Exercise 5. Alter the structure of one table by adding a new column.
+Alter Table Customers
+ADD PhoneNumber INT;
+--Exercise 6. Populate the value of the newly added column with relevant data.
+UPDATE Customers
+SET PhoneNumber =743341855
+WHERE Id = 6;
+UPDATE Customers
+SET PhoneNumber =728341855
+WHERE Id = 5;
+UPDATE Customers
+SET PhoneNumber =343418795
+WHERE Id = 4;
+UPDATE Customers
+SET PhoneNumber =743941351
+WHERE Id = 3;
+UPDATE Customers
+SET PhoneNumber =143641875
+WHERE Id = 2;
+UPDATE Customers
+SET PhoneNumber =793241865
+WHERE Id = 1;
+
+
+SELECT *
+FROM Customers;
+
+--Exercise 7. Display all the customers' information together with their orders.
+SELECT Customers.Name,Customers.Age,Customers.PhoneNumber,Orders.Details ,Products.Name
+FROM Customers
+JOIN Orders ON Customers.Id = Orders.Id
+JOIN Products ON Orders.Id = Products.Id ; 
+
+--Exercise 8. Display all the customers' information together with their orders and their addresses.
+SELECT Customers.Name,Customers.Age,Customers.PhoneNumber,Addresses.StreetName,Addresses.StreetNumber,Addresses.City,Orders.Details
+FROM Customers
+FULL OUTER JOIN Addresses On AddressId = Addresses.Id
+JOIN Orders ON Customers.Id = Orders.CustomerId;
```

Comment:
Really nice homework! Easy to read & well structured (with spaces, newlines, lowercase and uppercase). Keep up with the great work!